app/grandchallenge/evaluation/models.py

```diff
@@ -1519,7 +1520,7 @@ def save(self, *args, **kwargs):
                     send_action=False,
                 )
 
-    def create_evaluation(self):
+    def create_evaluation(self, *, additional_inputs=None):
```

Comment:
Should we explicitly require the caller setting `additional_inputs` to `None`? Do we check that the `additional_inputs` are set when they are required?

Reply:
Good idea. I made it a required input to the function now and added a check in `create_evaluation`.